src/Build/PublicAPI/netstandard/PublicAPI.Shipped.txt

```diff
@@ -991,8 +991,12 @@ Microsoft.Build.Execution.BuildParameters.InputResultsCacheFiles.get -> string[]
 Microsoft.Build.Execution.BuildParameters.InputResultsCacheFiles.set -> void
 Microsoft.Build.Execution.BuildParameters.Interactive.get -> bool
 Microsoft.Build.Execution.BuildParameters.Interactive.set -> void
-Microsoft.Build.Execution.BuildParameters.IsolateProjects.get -> bool
 Microsoft.Build.Execution.BuildParameters.IsolateProjects.set -> void
+Microsoft.Build.Execution.BuildParameters.IsolateProjects.get -> Microsoft.Build.Execution.IsolateProjects
```

Comment:
This part (`bool` to `IsolateProjects` enum) is a breaking change, so might not be allowed for compatibility.

Reply:
Addressed [here](https://github.com/dotnet/msbuild/pull/8249#issuecomment-1358019754).